config/initializers/default_content_type.rb

```diff
@@ -1,5 +1,9 @@
 require 'rack/default_content_type'
 
 Rails.application.configure do
-  config.middleware.use ::Rack::DefaultContentType
+  # We want to ensure requests have an expected content type
+  # before other middleware runs to make sure any body parsing
+  # attempts are handled correctly. So we add this middleware
+  # to the start of the Rack middleware chain.
+  config.middleware.insert_before(0, ::Rack::DefaultContentType)
```

Comment:
Here's one I consider important:

I'd like to see _everything_ that messes with the middleware chain in a single initializer.  This is a case where the benefit of "having a central view of all the magic/danger" far outweighs the cleanliness of separate files.

One place, clearly marked, that says:

"This is where all the nonstandard Rails stuff you might not expect is hapenning.  Read this to have a clear view of all the magic we've introduced for Conjur"

Reply:
To be clear, you want to change this in this PR, correct? The file itself already exists, this PR is currently only changing where the middleware is inserted.

Reply:
@jonahx @jtuttle , this is done. Double check that it is what you had in mind. Thanks!